python/iot-rest-api-sample.py

```diff
@@ -157,59 +228,52 @@
     'errorStackTraceIndex': 0,
     'severity': 'fatal',
   }]
-}]
+  beacon = [{}]
+  beacon[0]['deviceInfo'] = device_info
+  beacon[0]['versionInfo'] = version_info
+  beacon[0]['errorEvents'] = error_events
+  print("send error event")
+  send_beacon(beacon)
 
 
+# Send beacon to IoT Collector
 def send_beacon(beacon):
+  # Compress Payload
   out = io.BytesIO()
   with gzip.GzipFile(fileobj=out, mode='w') as f:
     json_str = json.dumps(beacon)
     json_bytes = json_str.encode('utf-8')
     f.write(json_bytes)
 
-  print('beacon: {}'.format(beacon))
+  if args.verbose:
+    print('beacon: {}'.format(beacon))
 
-  # validate beacon before sending (optional)
   r = requests.post(
-        iot['validateBeaconUrl'],
+        sendBeaconUrl,
         headers={
           'Content-Type': 'application/json',
-          'Content-Length': str(len(json_str)),
+          'Content-Length': str(len(out.getvalue())),
```

Comment:
just curious, doesn't post just do this for you?

Reply:
Yes, it does. Removed it.
DONE